apps/flair/test/integration/test_helper.exs

```diff
@@ -1,23 +1 @@
 ExUnit.start()
```

Comment:
can we just get rid of the whole integration subdirectory?

Reply:
Not yet. `scripts/travis-test.sh` is going to go to each app and run `mix test.integration`. That'll blow up if the integration subdirectory doesn't exist.

I didn't want to touch CI as part of this PR, so I'm just leaving the subdirectory.

Reply:
fair point; as long as it's just skips over it when it doesn't find any files it doesn't bug me.